packages.csv

```diff
@@ -0,0 +1,26 @@
+# Do not remove these comments
+# Format of the file : <Distro Name>,<Fuse Version, Architecture, File Type>,<Repo Name>,<Release Name(for deb repos only)>
+Ubuntu-18.04,azcopyAmdDeb,microsoft-ubuntu-bionic-prod-apt,bionic
+Ubuntu-20.04,azcopyAmdDeb,microsoft-ubuntu-focal-prod-apt,focal
+Ubuntu-22.04,azcopyAmdDeb,microsoft-ubuntu-jammy-prod-apt,jammy
+Ubuntu-22.04,azcopyArmDeb,microsoft-ubuntu-jammy-prod-apt,jammy
+Ubuntu-24.04,azcopyAmdDeb,microsoft-ubuntu-noble-prod-apt,noble
+Ubuntu-24.04,azcopyArmDeb,microsoft-ubuntu-noble-prod-apt,noble
+Debian-9.0,azcopyAmdDeb,microsoft-debian-stretch-prod-apt,stretch
+Debian-10.0,azcopyAmdDeb,microsoft-debian-buster-prod-apt,buster
+Debian-11.0,fuse3AmdDeb,microsoft-debian-bullseye-prod-apt,bullseye
```

Comment:
This needs to be corrected, `fuse3AmdDeb`

Reply:
corrected